chapters/functions.tex

```diff
@@ -171,7 +171,8 @@ \section{Function as a Specialized Class}\label{function-as-a-specialized-class}
 \item
   For initialization of local variables of a function see \cref{initialization-and-binding-equations-of-components-in-functions}).
 \item
-  Components of a function will inside the function behave as though they had discrete-time variability.
+  Components of a function follow special variability rules, see \cref{discrete-time-expressions} for details.
+  In particular the annotation \lstinline!GenerateEvents=true! makes this complicated.
```

Comment:
It would be good to avoid saying that things get complicated:
```suggestion
  Components of a function follow special variability rules where the \lstinline!GenerateEvents! annotation plays a central role, see \cref{discrete-time-expressions} for details.
```